docs/install/vendor-options.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Commercial Knative Offerings"
+linkTitle: "Knative Vendors"
+weight: 15
+type: "docs"
+---
+
+Knative has a rich community with many vendors participating, and many of those
+vendors offer commercial Knative offerings.  Please check with each of these
+vendors for what is or is not supported.
+
+<!-- TODO: Talk about criteria for inclusion here?  a la Conformance -->
+
+Here is a list of commercial Knative products (alphabetically):
+
+- [Google Cloud Run for Anthos](https://cloud.google.com/run/docs/gke/setup): <!-- TODO(ryangregg): add a description -->
+- IBM Kubernetes Service managed Knative add-on: <!-- TODO(duglin): add a description/link --> 
```

Comment:
@duglin  I'd like to include a short description of each of these (also please feel free to suggest a different name.  For IBM, I also need a link.

Reply:
- Name: Managed Knative for IBM Cloud Kubernetes Service
- link: https://cloud.ibm.com/docs/containers?topic=containers-serverless-apps-knative
- Desc: is a managed add-on for the [IBM Kubernetes Service](https://cloud.ibm.com/docs/containers?topic=containers-getting-started) that enables you to deploy and manage Knative based services on your own Kubernetes cluster.

Reply:
^^

Reply:
Perfect, thanks Doug!